packages/mui-codemod/src/deprecations/utils/movePropIntoSlotProps.js

```diff
@@ -3,50 +3,81 @@ import findComponentDefaultProps from '../../util/findComponentDefaultProps';
 import assignObject from '../../util/assignObject';
 import appendAttribute from '../../util/appendAttribute';
 
-function moveJsxPropIntoSlotProps(j, element, propName, slotName) {
+function moveJsxPropIntoSlotProps(j, element, propName, slotName, slotPropName) {
```

Comment:
Current version of `moveJsxPropIntoSlotProps` doesn't support moving a prop under specific name.

For example current version doesn't support below transformation

From

`<SpeedDialAction tooltipPlacement='top'`/>

To

`<SpeedDialAction slotProps={{tooltip:{placement:"top"}}}`/>

So i've made changes to support this